migration/capp-migration-service/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
According to the new policy, it should be WSO2 LLC. Please check other locations as well.

Reply:
Fixed.